inst/COPYRIGHTS

```diff
@@ -2,7 +2,7 @@ The package XLConnect includes other open source software components. The follow
 
```

Comment:
It really only "includes" (upfront) curvesapi these days and downloads others on-the-fly. We should probably update this statement.